File: 3sem/poo/Aula01-Codigo/C++/Exemplo04-Heranca/Terrestres.cpp
#include <string>
#include "Animal.cpp"

using namespace std;

class Terrestres : public Animal{
  public:    
    int iNrPatas;
    Terrestres(string saNome, int umNroPatas) : Animal(saNome){
        iNrPatas = umNroPatas;
    };

    int getiNrPatas() {
        return iNrPatas;
    };   
};

File: 3sem/poo/Aula01-Codigo/Java/Exemplo04-Heranca/src/exemplo4heranca/Terrestres.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */

package exemplo4heranca;

/**
 *
 * @author Junio
 */
public class Terrestres extends Animal{
    int iNrPatas;
    Terrestres(String saNome, int umNroPatas){
        super(saNome);
        iNrPatas = umNroPatas;
    }

    public int getiNrPatas() {
        return iNrPatas;
    }   
}
